### ISSUE TITLE
embedding notes via header distorts the order of elements

### ISSUE THREAD

**Issue by fnumatic** [MSzturc/obsidian-advanced-slides#153]
**Describe the bug**
The resulting html contains wrong ordered elements after embedding notes into slides via `[[file|header]]`

**Samples to Reproduce**
```
# slide test
---
![[bar embed#Foo]]
```
```
# bar embed

## Foo
- 1
- 2
- 3
- 4

extra line
```

results in 
![screen embed w heading](https://user-images.githubusercontent.com/1319530/197275995-c407181a-fb51-403f-8eec-d1d85a44e5c2.png)
![wrong order html](https://user-images.githubusercontent.com/1319530/197276613-dfe576cf-aa0a-4bf0-a681-bfef65387d19.png)

**Expected behavior**
It should be like
![screen embed raw](https://user-images.githubusercontent.com/1319530/197276957-be18ef66-da33-494a-9750-5342b98bf05d.png)
![right order html](https://user-images.githubusercontent.com/1319530/197276991-e56a2a80-bf1c-4791-a817-2b45fc8b7f52.png)

**Comment by fnumatic** [comment]
here are some logging trace
1. md direkt on slide
2. md via file embed
3. md via file#header embed

it seems that an empty line get's removed
```
plugin:obsidian-advanced-slides:318 merge & template: 

# slide test

---
## Foo
- 1
- 2
- 3
- 4

extra line

---
## Foo
- 1
- 2
- 3
- 4

extra line

---
## Foo
- 1
- 2
- 3
- 4
extra line


---
```

**Comment by MSzturc** [comment]
I was able to reproduced this issue. will be fixed in next version

**Comment by MSzturc** [comment]
will be fixed with 1.19
